scripts/deployment/README.md

```diff
@@ -25,6 +25,14 @@ Parameters of the `globals.json` file:
 - `useLedger`: a flag whether to use the hardware wallet (`true`) or proceed with the seed-phrase accounts (`false`);
 - `derivationPath`: a string with the derivation path;
 - `providerName`: a network type (see `hardhat.config.js` for the network configurations);
+- `serviceStakingParams`: a set of service staking parameters, which include:
+  - `rewardsPerSecond`: amount of token (in wei) per second credited to a service if the service is active enough. Assuming the maximum
```

Comment:
```suggestion
  - `rewardsPerSecond`: amount of token (in wei) per second credited to a service if the service is active enough.  The formula cn be obtained as follows`7days_rewards_per_service/7days_in_seconds` Assuming the maximum
```

Reply:
There's some issues with formatting, please check for spaces and names, as I am not sure how to correct that.

Reply:
let's ignore this. Later we can add more details if more info are required